class.jetpack.php

```diff
@@ -3181,6 +3181,8 @@ function admin_init() {
 			add_action( 'jetpack_notices', array( $this, 'alert_auto_ssl_fail' ) );
 		}
 
+		jetpack_tracks_get_identity( get_current_user_id() );
```

Comment:
This feels like a better home, so we're at least making sure that it's only being set if Jetpack is connected.  

https://github.com/Automattic/jetpack/blob/8440afcf9597d5396e40a4b7ebbb330a9d2f349a/class.jetpack-tracks.php#L15